manifests/config.pp

```diff
@@ -76,4 +79,67 @@
       group   => 'grafana',
     }
   }
+
+  # If grafana version is > 5.0.0, and the install method is package,
+  # repo, or archive, then use the provisioning feature. Dashboards
+  # and datasources are placed in
+  # /etc/grafana/provisioning/[dashboards|datasources].
+  # --dashboards--
+  if ((versioncmp($::grafana::version, '5.0.0') >= 0) and ($myprovision)) {
+    $pdashboards = $::grafana::provisioning_dashboards
```

Comment:
please use `$grafana::provisioning_dashboards` instead. There is no need for the absolute topscope anymore.